ui/components/Header.js

```diff
@@ -74,13 +74,13 @@ const styles = (theme) => ({
   appBarOnDrawerOpen: {
     backgroundColor: theme.palette.secondary.mainBackground,
     shadowColor: ' #808080',
-    zIndex: theme.zIndex.drawer + 1,
+    zIndex: 1100,
```

Comment:
We use an internal utility named `ziCalc` to set zIndex levels.
Would you mind using that instead of hardcoding 1100? :smile: 
This helps us to keep a more clear and easily recognizable track of zIndices.

Reply:
thanks @dragon-slayer875, but the result is not as expected, problem occurs on other side. Would you clarify me, Do we need to pas any args in `ziCalc()` in our scenario

https://github.com/meshery/meshery/assets/93466587/a0a49d15-4d3a-46fa-bbc8-c98e3fc70ccb

Reply:
A numerical value sir. :smile:

Reply:
@dragon-slayer875 , 'm surprised that removing the zIndex prop from the class works as expected. The default MUI navbar zIndex is applied without any hardcoded values or utils needed. Your suggestions please.

https://github.com/meshery/meshery/assets/93466587/0ef7ec0c-468d-4b89-8399-2850a474897c

Reply:
@dragon-slayer875 , seems to be it doesn't affect any other dashboard. 

https://github.com/meshery/meshery/assets/93466587/5d2322df-2a67-4d20-bc76-a1bbb4394082